SU-SOE/stanford_news
# D8 core 1906

# READY FOR REVIEW 

# Summary
- Adds a template for the dek to give it a flex-7-of-12 class.

# Review By (Date)
- May 1st?

# Urgency
- Low

# Steps to Test

1. Pull in changes.
2. Verify that the dek now has the flex-7-of-12 class.

# Affected Projects or Products
- Effects the news news article

# Associated Issues and/or People
- D8CORE-1906

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
